### PR TITLE
Fix incorrect formatting of code blocks in web help

### LINKED ISSUE #535: Web help examples are formatted incorrectly
Looks good in Zowe 1.18:
image

But something broke in Zowe 1.19:
image

### PR DESCRIPTION
**Pull request by t1m0thyj**
Fixes #535. The issue seems to have been caused by a breaking change in the `marked` package which was updated to fix vulnerabilities in #515.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+- BugFix: Fixed incorrect formatting of code blocks in web help. [#535](https://github.com/zowe/imperative/issues/535)
+
 ## `4.10.2`
 - Fix vulnerabilities by updating marked
```
Comment (zFernand0) [inline CHANGELOG.md:9]: Don't we have spaces between the version number and the list?
I know it doesn't really matter to GH though
(just trying to be annoying)  😋

**Review by zFernand0 — APPROVED** [review]
LGTM!

**Review by gejohnston — APPROVED** [review]
Good to go.

Remind me again, are we merging master into next after each such pull request into master?

**Comment by t1m0thyj** [comment]
> Remind me again, are we merging master into next after each such pull request into master?

Good reminder, I would have forgotten this 🙂 I believe the answer is yes.

**Comment by t1m0thyj** [comment]
Cherry picked fix made by @gejohnston in "next" branch for test that keeps failing because it expects 1000 and gets 999.

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**